Annotato/Annotato/Persistence/PersistenceService/LocalPersistenceService/CoreDataEntities/Extensions/AnnotationEntity+DataAccess.swift

```diff
@@ -0,0 +1,17 @@
+extension AnnotationEntity {
+    static var localDocumentEntityDataAccess: LocalDocumentEntityDataAccess {
+        LocalDocumentEntityDataAccess()
+    }
+
+    static var localAnnotationTextEntityDataAccess: LocalAnnotationTextEntityDataAccess {
+        LocalAnnotationTextEntityDataAccess()
+    }
+
+    static var localAnnotationHandwritingEntityDataAccess: LocalAnnotationHandwritingEntityDataAccess {
+        LocalAnnotationHandwritingEntityDataAccess()
+    }
+
+    static var localSelectionBoxEntityDataAccess: LocalSelectionBoxEntityDataAccess {
+        LocalSelectionBoxEntityDataAccess()
+    }
+}
```

Comment:
Is it possible to just initialise each time instead of storing in a static variable?